src/test/java/pages/CreateAnAccountPage.java

```diff
@@ -0,0 +1,172 @@
+    private WebElement submitButton;
+    @FindBy(id = "uniform-id_state")
+    private WebElement stateButton;
+
+    public CreateAnAccountPage(boolean openPageByURL) {
+        super(openPageByURL);
+    }
+
+    @Override
+    protected void openPage() {
+        driver.get("http://automationpractice.com/index.php?controller=authentication&back=my-account#account-creation");
+    }
+
+    @Override
+    protected boolean isPageOpened() {
+        try {
+            return submitButton.isDisplayed();
+        } catch (NoSuchElementException ex) {
+            return false;
+        }
+    }
+
+    private void setPersonalFirstName() {
+        personalFirstNameInput.sendKeys(ReadProperties.getInstance().getFirstName());
+    }
+
+    private void setPersonalLastName() {
+        personalLastNameInput.sendKeys(ReadProperties.getInstance().getLastName());
+    }
+
+    private void setPassword() {
+        passwordInput.sendKeys(ReadProperties.getInstance().getPassword());
+    }
+
+    private void setEmail() {
+        passwordInput.sendKeys(ReadProperties.getInstance().getEmail());
+    }
+
+    private void clickEmail() {
+        emailInput.click();
+    }
+
+    private void setAddressFirstName() {
+        addressFirstNameInput.sendKeys(ReadProperties.getInstance().getFirstName());
+    }
+
+    private void setAddressLastName() {
+        addressLastNameInput.sendKeys(ReadProperties.getInstance().getLastName());
+    }
+
+    private void setCompany() {
+        companyInput.sendKeys(ReadProperties.getInstance().getCompany());
+    }
+
+    private void setAddress() {
+        addressInput.sendKeys(ReadProperties.getInstance().getAddress());
+    }
+
+    private void setCity() {
+        cityInput.sendKeys(ReadProperties.getInstance().getCity());
+    }
+
+    private void setPostcode() {
+        postcodeInput.sendKeys(ReadProperties.getInstance().getZip());
+    }
+
+    private void setPhone() {
+        phoneMobileInput.sendKeys(ReadProperties.getInstance().getPhone());
+    }
+
+    private void clearAlias() {
+        aliasInput.clear();
+    }
+
+    private void setAlias() {
+        aliasInput.sendKeys(ReadProperties.getInstance().getAlias());
+    }
+
+    private void submitButton() {
+        submitButton.click();
+    }
+
+    private void stateButton() {
+        wait.until(ExpectedConditions.elementToBeClickable(stateButton)).click();
+    }
+
+    private void selectByIndexTitle(int index) {
+        List<WebElement> listTitle = driver.findElements(By.xpath("//input[@type ='radio']"));
+        for (WebElement element : listTitle) {
+            if (Integer.parseInt(element.getAttribute("value")) == index) {
+                element.click();
+            }
+        }
+    }
+
+    private void selectStateByName(String state) {
+        stateButton();
+        List<WebElement> listState = new ArrayList<>();
+        for (WebElement element : driver.findElements(By.xpath("//select[@name ='id_state']/option"))) {
```

Comment:
the same here, you can write xpath selector with required text, and select single element.